schema-dex-amm-extended.graphql

```diff
@@ -1,6 +1,6 @@
 # Subgraph Schema: DEX AMM (Extended)
 # Version: 2.0.0
```

Comment:
should probably update the version

Reply:
updated.

Reply:
Isn't this a major update bc it makes the api incompatible? https://semver.org/